stories/InteractionLayer.stories.js

```diff
@@ -470,3 +470,86 @@ export const regressionOnMouseUp = () => {
 regressionOnMouseUp.story = {
   name: 'Regression: onMouseUp',
 };
+
+const mockAreas = [
+  {
+    id: '1',
+    seriesId: '1',
+    color: 'lightblue',
+    start: {
+      xval: 1571577832254,
+      yval: 0.5,
+    },
+    end: {
+      xval: 1574169832254,
+      yval: 1,
+    },
+  },
+  {
+    id: '2',
+    seriesId: '2',
+    color: 'maroon',
+    start: {
+      xval: 1571577832254,
+      yval: 0.1,
+    },
+    end: {
+      xval: 1574169832254,
+      yval: 0.2,
+    },
+  },
+];
+
+export const AreasOnSeries = () => {
+  return (
+    <React.Fragment>
+      <DataProvider defaultLoader={staticLoader} timeDomain={staticXDomain}>
+        <Series id="1" color="steelblue" />
+        <Series id="2" color="maroon" />
+        <LineChart height={CHART_HEIGHT} areas={mockAreas} />
+      </DataProvider>
+    </React.Fragment>
+  );
+};
+AreasOnSeries.story = { name: 'Areas on series' };
+
+export const AreasNoEnd = () => {
+  return (
+    <React.Fragment>
+      <DataProvider defaultLoader={staticLoader} timeDomain={staticXDomain}>
+        <Series id="1" color="steelblue" />
+        <Series id="2" color="maroon" />
+        <LineChart
+          height={CHART_HEIGHT}
+          areas={[
+            {
+              id: '1',
+              seriesId: '1',
+              color: 'lightblue',
+              start: {
+                xval: 1571577832254,
+                yval: 0.5,
+              },
+            },
+          ]}
+        />
+      </DataProvider>
+    </React.Fragment>
+  );
+};
+AreasOnSeries.story = { name: 'Areas with no end' };
+
+export const AreasOnSeriesInCollection = () => {
```

Comment:
Does griff storybook have the knobs plugin? Would be super nice to get one of those set up for better testing of this. (or just use normal inputs)

Reply:
Yes I added knobs recently! I'm gonna go through all the storybooks, clean them up and sort them into groups better etc in a separate PR after this